number/Cargo.toml

```diff
@@ -19,6 +19,7 @@ csv = "1.3"
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive", "rc"] }
 serde_with = "3.6.1"
 schemars = { version = "0.8.16", features = ["preserve_order"]}
+ibig = "0.3.6"
```

Comment:
is `num-bigint` above still needed?

Reply:
Yes, because the large integer type of ark can only convert from num_bigint.